showcase/src/app/demo-ui/pages/dropdown/demo-dropdown-page.component.html

```diff
@@ -1,6 +1,27 @@
 <h1 class="mat-display-3" translate>SHOWCASE.DEMO.DROPDOWN.TITLE</h1>
 <section class="stark-section">
 	<h1 translate>SHOWCASE.DEMO.SHARED.EXAMPLE_VIEWER_LIST</h1>
+	<example-viewer
+		[extensions]="['HTML', 'TS']"
+		filesPath="dropdown/reactive-form-dropdown"
+		[exampleTitle]="'SHOWCASE.DEMO.DROPDOWN.REACTIVE_FORM' | translate"
+	>
+		<stark-dropdown
+			dropdownId="reactiveFormDropdown"
+			[options]="serviceDropdownOptions"
+			[dropdownFormControl]="serviceFormControl"
+			placeholder="SHOWCASE.DEMO.DROPDOWN.SELECT_SERVICE"
+			optionIdProperty="id"
+			optionLabelProperty="value"
+			required
+		>
+		</stark-dropdown>
+
+		<div *ngIf="selectedServiceFormControl">
```

Comment:
For this example, can you also demo how it can be disabled/enabled via the form control?

Reply:
Good idea 😊